Formula/openfast.rb

```diff
@@ -0,0 +1,27 @@
+class Openfast < Formula
+  desc "NREL-supported OpenFAST whole-turbine simulation code"
+  homepage "https://openfast.readthedocs.io"
+  url "https://github.com/rafmudaf/openfast/archive/v2.3.0.tar.gz"
+  sha256 "2332f4793ebfe88d2bc850d499239bac928c7cbfe036ba98c8314b677ee0cd31"
+
+  depends_on "cmake" => :build
+  depends_on "gcc"
+  depends_on "openblas"
+  depends_on :xcode
```

Comment:
Does it actually depend on all these at runtime? (and it can't work with apple clang?)

Reply:
Its fortran code so we are getting gfortran with gcc. Good question on openblas and Xcode, that may be redundant since I think we only need Xcode for Accelerate. I'll check that.